test/test.js

```diff
@@ -0,0 +1,49 @@
+var createS3UploadParams = require('../index').createS3UploadParams
+var reverse = require('../index').reverse
```

Comment:
Codacy has a fix for the issue: [Replace `'../index').reverse` with `"../index").reverse;`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
var reverse = require("../index").reverse;
```